Source/items.cpp

```diff
@@ -119,11 +119,11 @@ namespace {
 
 std::optional<CelSprite> itemanims[ITEMTYPES];
 
-enum anim_armor_id : uint8_t {
+enum class anim_armor_id : uint8_t {

```

Comment:
```suggestion
enum class PlayerArmorGraphics : uint8_t {
```

Reply:
I like it. 🙂 

I renamed the enum to `PlayerArmorGraphic`, cause we normally use singular for enum class.
I also renamed `anim_weapon_id` to `PlayerWeaponGraphic` so that both matches.

Reply:
Nice :)